### PR TITLE
Experimental cleanup in CSS-in-JS templates

### PR DESCRIPTION
**Pull request by jopdeklein**

### CODE REVIEW

File: src/transform/__testfixtures__/CSSinJS-broken-pattern.output.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const PriceElement = styled('div')(
+  ({ theme }) => css`
+    border-radius:${3}px`/*TOGGLE_REMOVE*/};
```
Comment (Gerrit88) [inline src/transform/__testfixtures__/CSSinJS-broken-pattern.output.js:3]: Are we really expecting this `/*TOGGLE_REMOVE*/` to be there after the cleanup?

**Review by Gerrit88 — APPROVED** [review]
Thanks, Jop.

Looks like it works well. I just have the one question, but I will approve in the mean time.

File: src/transform/__testfixtures__/CSSinJS-broken-pattern.output.js (hunk 1 above)
Comment (jopdeklein) [reply src/transform/__testfixtures__/CSSinJS-broken-pattern.output.js:3]: Yes, this example highlights the "broken" pattern, as a reminder for the patterns that don't work yet.

**Comment by jopdeklein** [comment]
Closing as it is properly fixed in a later version of JSCodeShift

**Pull request closed without merging**